markdown/developers/training_guide.md

```diff
@@ -0,0 +1,121 @@
+
+### Walkthrough
+
+#### Use this as a template.
+
+Use this file `https://github.com/nf-core/nf-co.re/markdown/developers/training_guide.md` as a template to guide you.
+
+1. Make a fork of the [nf-core website repository](https://github.com/nf-core/nf-co.re/).
+2. Make a new branch from the `master` branch in your fork and check it out. Name it after the guide you intend to submit. For example: `git checkout -b gitpod_training_guide`
+3. Make a copy of the `markdown/developers/training_guide.md` with an appropriate name to use as your skeleton.
+
+#### Add questions.
+
+Include between 1-3 questions your guide is intended to answer under the `##Questions` header above.
+
+#### Add objectives.
+
+Objectives are intended to describe the outcomes someone reading your guide should be able to do once they've read through it.
+The objectives also function as a checklist of things that should be part of the guide.
+
+#### Add a glossary.
+
+The glossary should define potentially unfamilar terms to a novice trying to use your guide for the first time.
+
+#### Prepare your reader.
+
+Make a checklist of what your reader needs to have done before they begin using this guide.
+
+#### Add the content.
+
+Under the `### Guide` section, add a step by step walkthrough of what you intend to demonstrate. Remember to include visual aids if it will help understanding.
+
+Initially it is unlikely there will be a bytesize talk to link to, so leave a message that one can be contributed.
+If a bytesize talk exists, then provide a link using this html under the `### Bytesize talk` section. Update the `src` URL to the correct youtube link.
+
+```html
+<div class="ratio ratio-16x9">
+    <iframe width="560" height="315" src="https://www.youtube.com/embed/xuNYATGFuw4" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+</div>
+```
+
+#### Add key points.
+
+Key points serve as a summary of what to do, and are important take home messages from the guide.
+
+#### Add supporting links.
+
+Provide links to where the reader can get additional support, or useful further reading.
+
+#### Format your document.
+
+Markdown must be formatted according to the rules set by a linting tool called _Prettier_ (<https://prettier.io/>).
+We recommend using an extension for your code editor to fix markdown for you automatically when you save a file - there are Prettier plugins [for most editors](https://prettier.io/docs/en/editors.html), including [VSCode](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode).
+Alternatively you can install and run the Prettier command line tool: `prettier -w .`
+
+#### Add your guide to the website navigation
+
+For now, new guides need to be added to the website navigation manually.
```

Comment:
Erm, @ewels, how is this done? I didn't see where to edit. I was expecting a `_site.yml` or something.

Reply:
Do you _really_ want to know? It's not pretty 😆  Basically just has to be manually added to the PHP for the header and footer:

https://github.com/nf-core/nf-co.re/blob/c6b04680ad8d6dfd6a3981d90f9dc51543b9acb4/includes/header.php#L171

https://github.com/nf-core/nf-co.re/blob/c6b04680ad8d6dfd6a3981d90f9dc51543b9acb4/includes/footer.php#L76

Generally speaking people write the files then either me or @mashehu add it to the nav 😆 

But yes, a `_site.yml` or something would be better. Or even just globbing the markdown directories and building the nav from that, using a YAML front-matter on the file to get stuff like the short title. Plenty of inspiration available from static site generators [like Hugo](https://gohugo.io/content-management/urls/#set-url-in-front-matter) etc.

Reply:
La la la, I'll take it out then, and let you guys take care of it. I don't think we should get new comers delving into this too.

Reply:
Agreed 👍🏻 I was hoping to get to this before the hackathon but it hasn't happened. I suspect that it might be the main thing I work on to be honest, as part of https://github.com/nf-core/nf-co.re/issues/352